interchaintest/misbehaviour_test.go

```diff
@@ -171,9 +174,18 @@ func TestScenarioMisbehaviourDetection(t *testing.T) {
 	// attempt to update client with duplicate header
 	b := cosmos.NewBroadcaster(t, chainA)
 
-	msg, err := clienttypes.NewMsgUpdateClient(clientID, newHeader, user.FormattedAddress())
+	m, ok := newHeader.(proto.Message)
+	require.True(t, ok)
+
+	any, err := codectypes.NewAnyWithValue(m)
```

Comment:
Since `any` is now an alias to `interface{}`, it's better to avoid naming variables `any`.

Reply:
Good point, changed this to `protoAny`